{
  "artifacts": [],
  "command": "optimize_function",
  "experiment": {
    "base_dir": "/home/freidanm/repos/pysgmcmc_experiments/pysgmcmc_experiments",
    "dependencies": [
      "numpy==1.14.5",
      "sacred==0.7.4"
    ],
    "mainfile": "experiment_wrapper.py",
    "name": "hpolib",
    "repositories": [],
    "sources": [
      [
        "__init__.py",
        "_sources/__init___d41d8cd98f00b204e9800998ecf8427e.py"
      ],
      [
        "experiment_wrapper.py",
        "_sources/experiment_wrapper_0b39f570ae9d23454a6ede1a6629eeea.py"
      ]
    ]
  },
  "fail_trace": [
    "Traceback (most recent call last):\n",
    "  File \"/home/freidanm/repos/pysgmcmc_experiments/experiment_env/lib/python3.5/site-packages/hpolib/abstract_benchmark.py\", line 76, in wrapper\n    wartortle = ConfigSpace.Configuration(self.configuration_space, squirtle)\n",
    "  File \"ConfigSpace/configuration_space.py\", line 999, in ConfigSpace.configuration_space.Configuration.__init__\n",
    "ValueError: Trying to set illegal value '[7.48971261]' forhyperparameter x0, Type: UniformFloat, Range: [-15.0, 10.0], Default: -2.5.\n",
    "\nThe above exception was the direct cause of the following exception:\n\n",
    "Traceback (most recent call last):\n",
    "  File \"/home/freidanm/repos/pysgmcmc_experiments/experiment_env/lib/python3.5/site-packages/sacred/config/captured_function.py\", line 46, in captured_function\n    result = wrapped(*args, **kwargs)\n",
    "  File \"hpolib_synthetic_functions.py\", line 70, in optimize_function\n    num_iterations=num_iterations,\n",
    "  File \"robo/robo/fmin/random_search.py\", line 72, in random_search\n    new_y = objective_function(new_x)\n",
    "  File \"/home/freidanm/repos/pysgmcmc_experiments/experiment_env/lib/python3.5/site-packages/hpolib/abstract_benchmark.py\", line 109, in __call__\n    return(self.objective_function(configuration, **kwargs)['function_value'])\n",
    "  File \"/home/freidanm/repos/pysgmcmc_experiments/experiment_env/lib/python3.5/site-packages/hpolib/abstract_benchmark.py\", line 79, in wrapper\n    'into a ConfigSpace.Configuration object') from e\n",
    "Exception: Error during the conversion of the provided into a ConfigSpace.Configuration object\n"
  ],
  "heartbeat": "2018-06-27T11:12:52.956302",
  "host": {
    "ENV": {},
    "cpu": "Intel(R) Xeon(R) CPU E5-2650 v2 @ 2.60GHz",
    "hostname": "metaex20",
    "os": [
      "Linux",
      "Linux-4.4.0-112-generic-x86_64-with-Ubuntu-16.04-xenial"
    ],
    "python_version": "3.5.2"
  },
  "meta": {
    "command": "optimize_function",
    "options": {
      "--beat_interval": null,
      "--capture": null,
      "--comment": null,
      "--debug": false,
      "--enforce_clean": false,
      "--file_storage": null,
      "--force": false,
      "--help": false,
      "--loglevel": null,
      "--mongo_db": null,
      "--name": null,
      "--pdb": false,
      "--print_config": false,
      "--priority": null,
      "--queue": false,
      "--sql": null,
      "--tiny_db": null,
      "--unobserved": false,
      "COMMAND": null,
      "UPDATE": [
        "benchmark=levy",
        "method=random_search"
      ],
      "help": false,
      "with": true
    }
  },
  "resources": [],
  "result": null,
  "start_time": "2018-06-27T11:12:52.339901",
  "status": "FAILED",
  "stop_time": "2018-06-27T11:12:52.960629"
}